apps/server/src/modules/management/console/database-management.console.ts

```diff
@@ -19,12 +20,20 @@ export class DatabaseManagementConsole {
 				required: false,
 				description: 'filter for a single <collection>',
 			},
+			{
+				flags: '-o, --onlyfactories',
+				required: false,
+				description: 'seed from factories only',
+			},
 		],
 		description: 'reset database collections with seed data from filesystem',
 	})
 	async seedCollections(options: Options): Promise<string[]> {
 		const filter = options?.collection ? [options.collection] : undefined;
-		const collections = await this.databaseManagementUc.seedDatabaseCollectionsFromFileSystem(filter);
+
+		const collections = options.onlyfactories
```

Comment:
Maybe with and and not with or to replace all what is possible and let the legacy still alive.